### DISCUSSION TITLE
Add a support for convolution with impulse response

### DISCUSSION THREAD

**Discussion by adrian-stepien** [bitgapp/eqMac#637] [Ideas]
On Windows, one might use EqualizerAPO to apply an FIR filter (e.g. designed in rePhase). As far as I am aware on Mac there is no such option and this might be the first tool to support that.

**Comment by mustevenplay** [comment]
Adding to this idea, I think it would be just amazing to have te possibility to rearrange the order of the effect chain. (Any. Both the native ones and the future VST ones that are yet to come)
